### ISSUE TITLE
patchBodyValidation should be resistent if body.Operations is not an array

### ISSUE THREAD

**Issue by soccermax** [thomaspoignant/scim-patch#289]
**Describe the bug**
In my unit-test suite I test my scim service against a various number of invalid patch requests. I just tested my test suite with this node package and encountered that the library crashes if operations is not an array. Sure this is a edge case and an invalid request but the package should be able to handle this. 

**To Reproduce**

Sample request:
```json
{
  "Operations": {
    "op": "add",
    "path": "members",
    "value": {}
  },
  "schemas": [
    "urn:ietf:params:scim:api:messages:2.0:PatchOp"
  ]
}
```

exception:
```
TypeError: body.Operations.forEach is not a function
    at Object.patchBodyValidation (/***/node_modules/scim-patch/src/scimPatch.ts:87:21)
```

**Comment by thomaspoignant** [comment]
@soccermax thanks for opening this issue.
I have not much time to work on it for now but if you want to open a pull request to fix it, it would be awesome.

I will get back to it as soon as I have some free time.

**Comment by thomaspoignant** [comment]
@soccermax a fix was released in the version `0.6.6`.
